src/tasks/installers/operator.ts

```diff
@@ -114,10 +116,15 @@ export class OperatorTasks {
         title: `Create CRD ${this.cheClusterCrd}`,
         task: async (ctx: any, task: any) => {
           const exist = await kube.crdExist(this.cheClusterCrd)
+          const yamlFilePath = ctx.resourcesPath + 'crds/org_v1_che_crd.yaml'
+
           if (exist) {
+            const checkCRD = await kube.isCRDEqual(this.cheClusterCrd, yamlFilePath)
+            if (!checkCRD) {
+              cli.error(`CRD read from ${yamlFilePath} are not compatible with existed one in cluster. Please update CRD.`)
```

Comment:
It is not possible to proceed the installation of Eclipse Che. The existed ${this.cheClusterCrd} is different from a new one. Please update it to continue the installation.